src/PrestaShopBundle/Resources/config/services/adapter/tax.yml

```diff
@@ -8,7 +8,8 @@ services:
   prestashop.adapter.tax.tax_computer:
     class: 'PrestaShop\PrestaShop\Adapter\Tax\TaxComputer'
     arguments:
-      - '@prestashop.adapter.tax_rules_group.repository.tax_rules_group_repository'
+      - '@=service("prestashop.adapter.legacy.context").getContext().language.id'
+      - '@prestashop.adapter.country.repository.country_repository'
```

Comment:
is this change introduce a BC break ?

Reply:
according to recent ADR https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md
![Screenshot from 2022-08-04 11-32-38](https://user-images.githubusercontent.com/31609858/182801970-df973e70-edb9-4ccb-85fa-393291a8464b.png)

As long as class instantiation is delegated to container, it is not a BC break. And thats the case (tax computer is a service defined in container services configuration and is not designed to be instantiated manually). So no BC break here.